src/components/pages/ParentBooking/ParentBookingCard.js

```diff
@@ -1,23 +1,31 @@
 import React from 'react';
 import { Card } from 'antd';
 
-const ParentBookingCard = ({ sessions }) => {
-  const { course, start_date, start_time, end_date, end_time } = sessions;
+const ParentBookingCard = ({ booking }) => {
```

Comment:
If you need to use the props' values you could destructure 'booking' here between these lines. 

ex: 

``` const { subject, title, ... etc} = {...booking}; ```